### PR TITLE
Five floor

### PR DESCRIPTION
**Pull request by tarasborov**
The fifth floor with stairs was added to the simulation world.

### CODE REVIEW

File: rs_gazebo/launch/HQ.launch [hunk 1]
```diff
@@ -3,8 +3,9 @@
   <arg name="use_sim_time" default="true" />
   <arg name="gui" default="true" />
   <arg name="headless" default="false" />
-  <!-- <arg name="world_name" default="$(find rs_gazebo)/worlds/ss_hq4.world" /> -->
- <arg name="world_name" default="empty.world" /> -
+  <!-- <arg name="world_name" default="$(find rs_gazebo)/worlds/ss_hq4and5.world" /> -->
+  <!-- <arg name="world_name" default="empty.world" /> -->
```
Comment (ldemk) [inline rs_gazebo/launch/HQ.launch]: remove unnecessary comments

File: rs_gazebo/models/ss_hq4/stairs_45/model.config [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<model>
+    <name>Untitled</name>
+    <version>1.0</version>
+    <sdf version="1.6">model.sdf</sdf>
+    <author>
+        <name></name>
+        <email></email>
+    </author>
```
Comment (ldemk) [inline rs_gazebo/models/ss_hq4/stairs_45/model.config]: add your email and yourself as author

File: rs_gazebo/models/ss_hq4/stairs_45/model.sdf [hunk 3]
```diff
@@ -0,0 +1,165 @@
+<?xml version='1.0'?>
+<sdf version='1.6'>
+  <model name='Untitled'>
```
Comment (ldemk) [inline rs_gazebo/models/ss_hq4/stairs_45/model.sdf]: Add some meaningful title to the model

File: rs_gazebo/worlds/ss_hq4and5.world [hunk 4]
```diff
@@ -0,0 +1,9889 @@
...
+        <visual name='Wall_10_Visual'>
+          <pose frame=''>0 0 1.25 0 -0 0</pose>
+          <geometry>
+            <box>
+              <size>1.75 0.15 2.5</size>
+            </box>
+          </geometry>
+          <material>
+            <script>
+              <uri>file://media/materials/scripts/gazebo.material</uri>
```
Comment (ldemk) [inline rs_gazebo/worlds/ss_hq4and5.world:204]: Are you sire it will work with absolute path?

File: rs_gazebo/worlds/stairs_only.world [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+      <pose frame=''>0 0 10 0 -0 0</pose>
+      <!--<diffuse>0.8 0.8 0.8 1</diffuse>-->
+      <!--<specular>0.2 0.2 0.2 1</specular>-->
+      <!--<attenuation>-->
+        <!--<range>1000</range>-->
+        <!--<constant>0.9</constant>-->
+        <!--<linear>0.01</linear>-->
+        <!--<quadratic>0.001</quadratic>-->
+      <!--</attenuation>-->
+      <!--<direction>-0.5 0.1 -0.9</direction>-->
```
Comment (ldemk) [inline rs_gazebo/worlds/stairs_only.world]: Remove the comments if they are not needed

File: rs_gazebo/worlds/stairs_only.world [hunk 6]
```diff
@@ -0,0 +1,240 @@
+<?xml version="1.0"?>
+<sdf version='1.6'>
+  <world name='default'>
```
Comment (ldemk) [inline rs_gazebo/worlds/stairs_only.world]: Change the name to meaningful title

File: rs_inverse/scripts/env_tester.py [hunk 7]
```diff
@@ -143,33 +146,33 @@ def callback_gait(data):
...
+pitch_inst = 0.
+yaw_inst = 0.
+vx_inst = 0.
+vy_inst = 0.
+vz_inst = 0.
+wz_inst = 0.
+search_index = -1
 
 
 def callback_model(data):
```
Comment (ldemk) [inline rs_inverse/scripts/env_tester.py:166]: Much better is to make it a part of the class and then you might omit the global variables declaration

**Review by ldemk — CHANGES_REQUESTED** [review]

File: rs_gazebo/launch/HQ.launch (hunk 1 above)
Comment (tarasborov) [reply rs_gazebo/launch/HQ.launch]: Done in 89ad8b4f29d0c4017397ebd86242b846ef349683

File: rs_gazebo/models/ss_hq4/stairs_45/model.sdf (hunk 3 above)
Comment (tarasborov) [reply rs_gazebo/models/ss_hq4/stairs_45/model.sdf]: Done in aeeefd33c02f0487cc1a09a8ad16bda8167f326c

File: rs_inverse/scripts/env_tester.py (hunk 7 above)
Comment (tarasborov) [reply rs_inverse/scripts/env_tester.py:166]: It was done in newer branch

**Review by tarasborov — COMMENTED** [review]
Done

File: rs_gazebo/worlds/stairs_only.world (hunk 6 above)
Comment (tarasborov) [reply rs_gazebo/worlds/stairs_only.world]: Done in ee04ad38f434ebae9fed92d10aa76bd8890ce770

File: rs_gazebo/models/ss_hq4/stairs_45/model.config (hunk 2 above)
Comment (tarasborov) [reply rs_gazebo/models/ss_hq4/stairs_45/model.config]: Done in aeeefd33c02f0487cc1a09a8ad16bda8167f326c

**Review by tarasborov — COMMENTED** [review]
Have done

File: rs_gazebo/worlds/stairs_only.world (hunk 5 above)
Comment (tarasborov) [reply rs_gazebo/worlds/stairs_only.world]: Done

File: rs_gazebo/worlds/ss_hq4and5.world (hunk 4 above)
Comment (tarasborov) [reply rs_gazebo/worlds/ss_hq4and5.world:204]: It works. This was in origin file ss_hq4

**Review by ldemk — APPROVED** [review]

**Pull request merged**